torch/linalg/__init__.py

```diff
@@ -362,18 +373,18 @@
              non-zero number produces another set of valid eigenvectors of the matrix.
              In this implmentation, the returned eigenvectors are normalized to have norm
              `1` and largest real component.
-""".format(**common_notes) + r"""
 
-.. warning:: This function assumes that :attr:`A` is `diagonalizable`_ (e.g. when all the
-             eigenvalues are different). If it is not diagonalizable, the returned
-             eigenvalues will be correct but :math:`A \neq V \operatorname{diag}(\Lambda)V^{-1}`.
-
-.. warning:: This function does not yet support autograd.
+.. warning:: Gradients computed using `V` will only be finite when :attr:`A` does not have repeated eigenvalues.
```

Comment:
I completely agree. The problem is that, for example, in this case, it was parsing the `\neq` as "\n" and was causing a `flake8` error. Even more, this way requires a specific split for every function so that no curly braces are in an area with a `common_string`...

Doing it this other way we keep the changes local, given that there are these "global" problems.